patches/chrome-browser-ui-views-location_bar-location_bar_view.cc.patch

```diff
@@ -11,11 +11,11 @@ index c5909031e62360b74de940c3e1fdc4d2ed51ad96..30aa96ebaa3a0b94b8e30a70d2e86cb8
    if (!IsInitialized())
      return;
  
-@@ -544,6 +544,11 @@ void LocationBarView::Layout() {
+@@ -550,6 +550,11 @@ void LocationBarView::Layout() {
      }
    };
  
-+  if (right_most && right_most->visible())
++  if (right_most && right_most->GetVisible())
```

Comment:
no need for follow-up here, but when you come across these multiline patches please convert them to `DEFINE`

Reply:
Addressed in d9b7bb5